packages/replay/src/eventBuffer/EventBufferCompressionWorker.ts

```diff
@@ -87,9 +85,9 @@ export class EventBufferCompressionWorker implements EventBuffer {
     this.hasCheckout = false;
 
     // We do not wait on this, as we assume the order of messages is consistent for the worker
-    this._worker.postMessage('clear').then(null, e => {
-      DEBUG_BUILD && logger.warn('[Replay] Clearing the event buffer failed.', e);
-    });
+    // This should never reject
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    this._worker.postMessage('clear');
```

Comment:
sadly, this _can_ reject 😬

Reply:
Alrighty, does this work? 06d041a53dceecd171282f90d0860a782ba03a57